docs/pipeline.rst

```diff
@@ -173,6 +173,18 @@ intent_classifier_sklearn
     to other classifiers it also provides rankings of the labels that did not "win". The spacy intent classifier
     needs to be preceded by a featurizer in the pipeline. This featurizer creates the features used for the classification.
 
+intent_featurizer_regex
```

Comment:
I can see two sensible solutions, 

1. keep others as `intent_featurizer_x` and call this either `featurizer_regex` or `intent_entity_featurizer_regex`

2. call everything `featurizer_x` and rely on docs to explain the difference